driver/ppm_flag_helpers.h

```diff
@@ -127,7 +127,11 @@ static __always_inline u32 open_modes_to_scap(unsigned long flags,
 					      unsigned long modes)
 {
 #ifdef UDIG
+#ifdef O_TMPFILE
```

Comment:
Since this file is later imported in userspace only in case of __linux__, do we really need all these ifdefs?